app/src/main/java/org/wikipedia/notifications/NotificationEditorTasksHandler.java

```diff
@@ -0,0 +1,91 @@
+                Prefs.setEditActionTranslateDescriptionsUnlocked(true);
+                maybeShowTranslateDescriptionUnlockNotification(context);
+                eventToDispatch = new EditorTaskUnlockEvent(numTargetsPassed);
+            }
+        } else if (numTargetsPassed > 0) {
+            if (!Prefs.isActionEditDescriptionsUnlocked()) {
+                Prefs.setActionEditDescriptionsUnlocked(true);
+                Prefs.setShowActionFeedIndicator(true);
+                Prefs.setShowEditMenuOptionIndicator(true);
+                maybeShowEditDescriptionUnlockNotification(context);
+                eventToDispatch = new EditorTaskUnlockEvent(numTargetsPassed);
+            }
+            if (Prefs.isEditActionTranslateDescriptionsUnlocked()) {
+                Prefs.setEditActionTranslateDescriptionsUnlocked(false);
+            }
+        } else {
+            if (Prefs.isActionEditDescriptionsUnlocked()) {
+                Prefs.setActionEditDescriptionsUnlocked(false);
+            }
+            if (Prefs.isEditActionTranslateDescriptionsUnlocked()) {
+                Prefs.setEditActionTranslateDescriptionsUnlocked(false);
+            }
+        }
+
+        if (eventToDispatch != null) {
+            WikipediaApp.getInstance().getBus().post(eventToDispatch);
+        }
+    }
+
+    private static void maybeShowEditDescriptionUnlockNotification(@NonNull Context context) {
+        if (!WikipediaApp.getInstance().isAnyActivityResumed()) {
+            Intent intent = AddTitleDescriptionsActivity.Companion.newIntent(context, Constants.InvokeSource.EDIT_FEED_TITLE_DESC);
+            NotificationCompat.Builder builder = NotificationPresenter.getDefaultBuilder(context);
+            PendingIntent pendingIntent = PendingIntent.getActivity(context, 1, intent, PendingIntent.FLAG_UPDATE_CURRENT);
+            builder.addAction(0, context.getString(R.string.title_description_get_started), pendingIntent);
+            NotificationPresenter.showNotification(context, 0, context.getString(R.string.description_edit_task_unlock_title),
+                    context.getString(R.string.description_edit_task_unlock_body),
+                    context.getString(R.string.description_edit_task_unlock_body),
+                    R.drawable.ic_mode_edit_white_24dp, R.color.accent50, intent);
+        }
+    }
+
+    private static void maybeShowTranslateDescriptionUnlockNotification(@NonNull Context context) {
+        if (!WikipediaApp.getInstance().isAnyActivityResumed()) {
+            Intent intent = AddTitleDescriptionsActivity.Companion.newIntent(context, Constants.InvokeSource.EDIT_FEED_TRANSLATE_TITLE_DESC);
+            NotificationCompat.Builder builder = NotificationPresenter.getDefaultBuilder(context);
+            PendingIntent pendingIntent = PendingIntent.getActivity(context, 1, intent, PendingIntent.FLAG_UPDATE_CURRENT);
+            builder.addAction(0, context.getString(R.string.title_description_get_started), pendingIntent);
```

Comment:
Noticed that the `builder` should be sent into the `NotificationPresenter.showNotification()`.

Reply:
Whoops! You are correct.